### PR TITLE
Typo3

### PR DESCRIPTION
**Pull request by fushi-gg**
no u

### CODE REVIEW

File: src/game/server/gamecontext.h [hunk 1]
```diff
@@ -220,6 +220,8 @@ class CGameContext : public IGameServer
 	char m_aVoteCommand[VOTE_CMD_LENGTH];
 	char m_aVoteReason[VOTE_REASON_LENGTH];
 	int m_NumVoteOptions;
+	int m_Ragequit;
```
Comment (GolezTrol) [inline src/game/server/gamecontext.h]: This shite needs to move to pplayer. Theze variables are for the whole game.

Comment (GolezTrol) [reply src/game/server/gamecontext.h]: I also think you need to initialize m_Ragequit to 0 somewhere (player constructor?)

File: src/game/server/gamecontext.cpp [hunk 2]
```diff
@@ -713,16 +713,32 @@ void CGameContext::OnClientConnected(int ClientID)
...
 {
 	m_pController->t_stats->on_drop(ClientID, pReason);
-	
-	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force) return false;
 
+	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force)
+	{
+		if(m_Ragequit <= 1)
+			m_Ragequit++;
+		if(m_Ragequit == 1)
```
Comment (GolezTrol) [inline src/game/server/gamecontext.cpp]: Should this be 1? Maybe a higher number depending on the time?

**Review by GolezTrol — CHANGES_REQUESTED** [review]

File: src/game/server/gamecontext.cpp [hunk 3]
```diff
@@ -712,17 +712,35 @@ void CGameContext::OnClientConnected(int ClientID)
 
 bool CGameContext::OnClientDrop(int ClientID, const char *pReason, bool Force)
 {
+	CPlayer *pPlayer = m_apPlayers[ClientID];
 	m_pController->t_stats->on_drop(ClientID, pReason);
-	
-	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force) return false;
 
+	if (m_apPlayers[ClientID]->GetCharacter() && m_apPlayers[ClientID]->GetCharacter()->IsFreezed() && !m_pController->IsGameOver() && !Force)
```
Comment (GolezTrol) [inline src/game/server/gamecontext.cpp]: You could use pPlayer here too, if you got it anyway...

**Review by GolezTrol — APPROVED** [review]

File: src/game/server/gamecontext.cpp (hunk 2 above)
Comment (fushi-gg) [reply src/game/server/gamecontext.cpp]: no u

File: src/game/server/gamecontext.h (hunk 1 above)
Comment (fushi-gg) [reply src/game/server/gamecontext.h]: yes and yes

File: src/game/server/gamecontext.cpp (hunk 3 above)
Comment (fushi-gg) [reply src/game/server/gamecontext.cpp]: oggi

**Comment by GolezTrol** [comment]
Awesomesauce!

**Review by nobody-mb — APPROVED** [review]
lmao

**Pull request merged**